File: Python/interactions/click_pagination.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://example.com")
    for _ in range(3):
        page.click("a.next-page")
        page.wait_for_timeout(1000)
    print(page.content())
    browser.close()

File: NodeJS/interactions/click_pagination.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  for (let i = 0; i < 3; i++) {
    await page.click('a.next-page');
    await page.waitForTimeout(1000);
  }
  console.log(await page.content());
  await browser.close();
})();
